src/stores/orderStore.ts

```diff
@@ -52,19 +51,15 @@ class OrderStore extends BaseStore {
   placeOrder = async (orderType: string, body: any) => {
     switch (orderType) {
       case OrderType.Market:
-        return (
-          this.api
-            .placeMarket(body)
-            // tslint:disable-next-line:no-empty
-            .then(() => {}, this.orderPlacedUnsuccessfully)
-        );
+        return this.api
+          .placeMarket(body)
+          .then(this.orderPlacedSuccessfully, this.orderPlacedUnsuccessfully)
+          .then(() => Promise.resolve());
       case OrderType.Limit:
-        return (
-          this.api
-            .placeLimit(body)
-            // tslint:disable-next-line:no-empty
-            .then(() => {}, this.orderPlacedUnsuccessfully)
-        );
+        return this.api
+          .placeLimit(body)
+          .then(this.orderPlacedSuccessfully, this.orderPlacedUnsuccessfully)
```

Comment:
As I can see below - you've merged rest handling for cancelling order. So placing limit should be merged as well